src/query/list_file.c

```diff
@@ -6760,3 +6770,11 @@ qfile_update_qlist_count (THREAD_ENTRY * thread_p, const QFILE_LIST_ID * list_p,
     }
 #endif // SERVER_MODE
 }
+
+int
+qfile_get_list_cache_number_of_entries (THREAD_ENTRY * thread_p, int ht_no)
```

Comment:
Because this function doesn't access thread-specific data, THREAD_ENTRY  is not needed

Reply:
I fixed it to remove the argument.